src/erlcloud_elb.erl

```diff
@@ -45,6 +48,14 @@
         "/DescribeLoadBalancerPoliciesResponse/DescribeLoadBalancerPoliciesResult/PolicyDescriptions/member").
 -define(DESCRIBE_ELB_POLICY_TYPE_PATH, 
         "/DescribeLoadBalancerPolicyTypesResponse/DescribeLoadBalancerPolicyTypesResult/PolicyTypeDescriptions/member").
+-define(DESCRIBE_ELBS_TAGS_PATH,
+        "/DescribeTagsResponse/DescribeTagsResult/TagDescriptions/member").
+-define(DESCRIBE_ELBS_TAGS_PATH_TOKEN,
+        "/DescribeTagsResponse/DescribeTagsResult/TagDescriptions/NextMarker").
+
+-type paged_result() :: {ok, term()} | {{paged, string()}, term()} |
```

Comment:
Could you please change paged output format to `{ok, Results, Marker}` as [readme suggests](https://github.com/erlcloud/erlcloud#best-practices)